packages/js/data/src/onboarding/actions.ts

```diff
@@ -548,6 +548,28 @@ export function* installAndActivatePluginsAsync(
 	}
 }
 
+export function* updateStoreCurrencyAndMeasurementUnits( countryCode: string ) {
+	yield setIsRequesting( 'updateStoreCurrencyAndMeasurementUnits', true );
+
+	try {
+		const results = yield apiFetch( {
```

Comment:
I have a PR that enables TS checks on `@woocommerce/data` package https://github.com/woocommerce/woocommerce/pull/55621


This morning I pulled new changes and this line triggers a TS build error:

```ts
'yield' expression implicitly results in an 'any' type because its containing generator lacks a return-type annotation.ts(7057)
```

Could you update this line, specifying the type of the `results` variable?

Reply:
Good catch @gigitux! apiFetch takes a generic type param which I think is ~= or maybe just slightly type-safer than specifying the variable type - it will catch the case where apiFetch's return type changes (unlikely but we're splitting hairs here), when specifying the variable type will just override it.

apiFetch type looks like: `apiFetch<T>(options) : Promise<T>`

Reply:
This is actually @moon0326 's PR but I've created the fix PR here: https://github.com/woocommerce/woocommerce/pull/55689